File: Dante1/Module1.07/Zadanie7.01/main.c
#include <stdio.h>
#include <math.h>

int test1(int a);

int main()
{
    int number;

    printf("Enter radix: ");
    int check = scanf("%i",&number);
    if(check != 1)
    {
        printf("Incorrect input\n");
        return 1;
    }


    printf("Result is: %i", test1(number));
    return 0;
}

int test1(int a)
{
    return a;
}

File: Dante1/Module1.07/Zadanie7.01/main.rs
fn test1(a: i32) -> i32
{
    a
}

fn main()
{
    let mut input: String = String::new();

    println!("Enter: ");

    std::io::stdin().read_line(&mut input).expect("Incorrect input");
    let number: i32 = input.trim().parse().expect("Incorrect parsing");

    println!("Result is: {}", test1(number));
}
